step.yml

```diff
@@ -0,0 +1,38 @@
+---
+title: Generate Xcode html report
+summary: The Step converts xcresult summaries to html reports.
+description: |-
+  Fill out.
```

Comment:
Is it intentionally left here?

Reply:
It was intentional but now I have filled out the whole step.yml properly.